web/packages/teleport/src/UnifiedResources/Resources.tsx

```diff
@@ -139,40 +125,33 @@ export function Resources() {
         pathname={pathname}
         replaceHistory={replaceHistory}
       />
-      {attempt.status === 'failed' && (
-        <ErrorMessage message={attempt.statusText} />
-      )}
       <ResourcesContainer gap={2}>
-        {fetchedData.agents.map((agent, i) => (
-          <ResourceCard key={i} onLabelClick={onLabelClick} resource={agent} />
+        {resources.map((res, i) => (
+          <ResourceCard key={i} onLabelClick={onLabelClick} resource={res} />
```

Comment:
We can't use index as key here as these elements will change order. I thought we added work to add an `id` to every `makeXResource` function but I don't know where that went. Can we add that so these can be a unique key?

Reply:
Yes, sure, will do